common/src/main/java/com/revenuecat/purchases/common/networking/ETagManager.kt

```diff
@@ -92,7 +92,7 @@ class ETagManager(
     }
 
     @Synchronized
-    internal fun clearCaches() {
+    fun clearCaches() {
```

Comment:
All the methods in `ETagManager` are `internal`. I'm not sure of the reason, I guess we wanted to only use it from the `common` module but still create it from a different module... Still, I think this is ok. Lmk if you think otherwise!

Reply:
Yes, probably to keep them in this module, I think it's safe to expose them